installers/rpm-code/codes/common.sh

```diff
@@ -59,6 +84,9 @@ common_main() {
         libffi-devel
         libtool
         llvm-libs
+        # https://github.com/radiasoft/devops/issues/225
+        # TODO(e-carlin):  add back in
+        # mongodb-org-server
```

Comment:
Because adding the mongo repo file didn't work so when the installer got here mongo couldn't be found and it would error out. I left it to remind myself of what was going on.